revenue_app/tests.py

```diff
@@ -310,3 +347,10 @@ def test_organizer_transactions_view_returns_302_when_not_logged(self):
         response = client.get(URL)
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, '/accounts/login/?next={}'.format(URL))
+
+    def test_transactions_search_view_returns_302_when_not_logged(self):
+        email = 'arg_domain@superdomain.org.ar'
+        URL = '/transactions/search/'
```

Comment:
maybe we could find a place to put all these URLs, and reuse them in the future.

Reply:
We could put the URLs on the setUp, if that's ok